docs/samples/shards/General/Await/1.edn

```diff
@@ -0,0 +1,21 @@
+;; Asynchronous execution of shards (using keyword `Await`)
+(defmesh mesh)
+
+(defloop await-wire
+(Await
+  ;:Shards
+  (->
+    (Msg "Await worker1")
+    ;; printed 1st => [info] [2022-08-07 16:35:31.636] [T-29916] [logging.cpp::98] [await-wire] Await worker1
```

Comment:
I wouldn't put the log here since every time we generate the doc, it would be different.
Just keep "printed 1st".

Reply:
Comments rewritten to better explain the expected order of logging (of messages) and the reason for this.